packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -1210,6 +1240,12 @@ class SearchBar extends StatefulWidget {
   /// {@macro flutter.widgets.editableText.autofocus}
   final bool autoFocus;
 
+  /// The type of action button to use for the keyboard.
+  final TextInputAction? textInputAction;
+
+  /// The type of keyboard to use for editing the text.
```

Comment:
Same here and the properties for `SearchAnchor`

Reply:
Done 👍✅